app/services/migration/migrators/user.rb

```diff
@@ -0,0 +1,34 @@
+module Migration::Migrators
+  class User < Base
+    def call
+      migrate(ecf_users, :user) do |ecf_user|
```

Comment:
Are we going to match on other attributes as part of this PR, like `trn` and `get_an_identity_id`? and how long does it take to run this migration?

Reply:
I think matching on just `ecf_id` for now might be the first step, but interested in the numbers and failures returned, before any data cleanup, also the length as you mentioned